## -------------------------- filteredExpr10_Koh ---------------------------- ##
dfSCEFilteredExpr10Koh <- data.frame(
  Title = "sceFilteredExpr10Koh",
  Description = "Gene counts for scRNA-seq data set of in vitro cultured H7 embryonic stem cells (WiCell) and H7-derived downstream early mesoderm progenitors. The top 10% genes with the highest average expression across all cells are retained.",
  BiocVersion = "3.7",
  Genome = "GRCh38",
  SourceType = "FASTQ",
  SourceUrl = "https://www.ncbi.nlm.nih.gov/sra?term=SRP073808",
  SourceVersion = NA,
  Species = "Homo sapiens",
  TaxonomyId = "9606",
  Coordinate_1_based = NA,
  DataProvider = "SRA (SRP073808), Koh et al. (2016)",
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "SingleCellExperiment",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/sce_filteredExpr10_Koh.rds",
  stringsAsFactors = FALSE
)

dfResFilteredExpr10Koh <- data.frame(
  Title = "clusteringSummaryFilteredExpr10Koh",
  Description = "Clustering results for sceFilteredExpr10Koh",
  BiocVersion = "3.7",
  Genome = "GRCh38",
  SourceType = NA,
  SourceUrl = NA,
  SourceVersion = NA,
  Species = "Homo sapiens",
  TaxonomyId = "9606",
  Coordinate_1_based = NA,
  DataProvider = NA,
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "data.frame",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/clustering_summary_filteredExpr10_Koh.rds",
  stringsAsFactors = FALSE
)

## -------------------------- filteredHVG10_Koh ----------------------------- ##
dfSCEFilteredHVG10Koh <- data.frame(
  Title = "sceFilteredHVG10Koh",
  Description = "Gene counts for scRNA-seq data set of in vitro cultured H7 embryonic stem cells (WiCell) and H7-derived downstream early mesoderm progenitors. The top 10% most highly variable genes are retained.",
  BiocVersion = "3.7",
  Genome = "GRCh38",
  SourceType = "FASTQ",
  SourceUrl = "https://www.ncbi.nlm.nih.gov/sra?term=SRP073808",
  SourceVersion = NA,
  Species = "Homo sapiens",
  TaxonomyId = "9606",
  Coordinate_1_based = NA,
  DataProvider = "SRA (SRP073808), Koh et al. (2016)",
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "SingleCellExperiment",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/sce_filteredHVG10_Koh.rds",
  stringsAsFactors = FALSE
)

dfResFilteredHVG10Koh <- data.frame(
  Title = "clusteringSummaryFilteredHVG10Koh",
  Description = "Clustering results for sceFilteredHVG10Koh",
  BiocVersion = "3.7",
  Genome = "GRCh38",
  SourceType = NA,
  SourceUrl = NA,
  SourceVersion = NA,
  Species = "Homo sapiens",
  TaxonomyId = "9606",
  Coordinate_1_based = NA,
  DataProvider = NA,
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "data.frame",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/clustering_summary_filteredHVG10_Koh.rds",
  stringsAsFactors = FALSE
)

## ------------------------- filteredM3Drop10_Koh --------------------------- ##
dfSCEFilteredM3Drop10Koh <- data.frame(
  Title = "sceFilteredM3Drop10Koh",
  Description = "Gene counts for scRNA-seq data set of in vitro cultured H7 embryonic stem cells (WiCell) and H7-derived downstream early mesoderm progenitors. The top 10% genes ranked by z-scores from the M3Drop package are retained.",
  BiocVersion = "3.7",
  Genome = "GRCh38",
  SourceType = "FASTQ",
  SourceUrl = "https://www.ncbi.nlm.nih.gov/sra?term=SRP073808",
  SourceVersion = NA,
  Species = "Homo sapiens",
  TaxonomyId = "9606",
  Coordinate_1_based = NA,
  DataProvider = "SRA (SRP073808), Koh et al. (2016)",
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "SingleCellExperiment",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/sce_filteredM3Drop10_Koh.rds",
  stringsAsFactors = FALSE
)

dfResFilteredM3Drop10Koh <- data.frame(
  Title = "clusteringSummaryFilteredM3Drop10Koh",
  Description = "Clustering results for sceFilteredM3Drop10Koh",
  BiocVersion = "3.7",
  Genome = "GRCh38",
  SourceType = NA,
  SourceUrl = NA,
  SourceVersion = NA,
  Species = "Homo sapiens",
  TaxonomyId = "9606",
  Coordinate_1_based = NA,
  DataProvider = NA,
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "data.frame",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/clustering_summary_filteredM3Drop10_Koh.rds",
  stringsAsFactors = FALSE
)

## ------------------------ filteredExpr10_KohTCC --------------------------- ##
dfSCEFilteredExpr10KohTCC <- data.frame(
  Title = "sceFilteredExpr10KohTCC",
  Description = "TCC counts for scRNA-seq data set of in vitro cultured H7 embryonic stem cells (WiCell) and H7-derived downstream early mesoderm progenitors. The top 10% genes with the highest average expression across all cells are retained.",
  BiocVersion = "3.7",
  Genome = "GRCh38",
  SourceType = "FASTQ",
  SourceUrl = "https://www.ncbi.nlm.nih.gov/sra?term=SRP073808",
  SourceVersion = NA,
  Species = "Homo sapiens",
  TaxonomyId = "9606",
  Coordinate_1_based = NA,
  DataProvider = "SRA (SRP073808), Koh et al. (2016)",
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "SingleCellExperiment",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/sce_filteredExpr10_KohTCC.rds",
  stringsAsFactors = FALSE
)

dfResFilteredExpr10KohTCC <- data.frame(
  Title = "clusteringSummaryFilteredExpr10KohTCC",
  Description = "Clustering results for sceFilteredExpr10KohTCC",
  BiocVersion = "3.7",
  Genome = "GRCh38",
  SourceType = NA,
  SourceUrl = NA,
  SourceVersion = NA,
  Species = "Homo sapiens",
  TaxonomyId = "9606",
  Coordinate_1_based = NA,
  DataProvider = NA,
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "data.frame",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/clustering_summary_filteredExpr10_KohTCC.rds",
  stringsAsFactors = FALSE
)

## ------------------------ filteredHVG10_KohTCC ---------------------------- ##
dfSCEFilteredHVG10KohTCC <- data.frame(
  Title = "sceFilteredHVG10KohTCC",
  Description = "TCC counts for scRNA-seq data set of in vitro cultured H7 embryonic stem cells (WiCell) and H7-derived downstream early mesoderm progenitors. The top 10% most highly variable genes are retained.",
  BiocVersion = "3.7",
  Genome = "GRCh38",
  SourceType = "FASTQ",
  SourceUrl = "https://www.ncbi.nlm.nih.gov/sra?term=SRP073808",
  SourceVersion = NA,
  Species = "Homo sapiens",
  TaxonomyId = "9606",
  Coordinate_1_based = NA,
  DataProvider = "SRA (SRP073808), Koh et al. (2016)",
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "SingleCellExperiment",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/sce_filteredHVG10_KohTCC.rds",
  stringsAsFactors = FALSE
)

dfResFilteredHVG10KohTCC <- data.frame(
  Title = "clusteringSummaryFilteredHVG10KohTCC",
  Description = "Clustering results for sceFilteredHVG10KohTCC",
  BiocVersion = "3.7",
  Genome = "GRCh38",
  SourceType = NA,
  SourceUrl = NA,
  SourceVersion = NA,
  Species = "Homo sapiens",
  TaxonomyId = "9606",
  Coordinate_1_based = NA,
  DataProvider = NA,
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "data.frame",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/clustering_summary_filteredHVG10_KohTCC.rds",
  stringsAsFactors = FALSE
)

## ----------------------- filteredM3Drop10_KohTCC -------------------------- ##
dfSCEFilteredM3Drop10KohTCC <- data.frame(
  Title = "sceFilteredM3Drop10KohTCC",
  Description = "TCC counts for scRNA-seq data set of in vitro cultured H7 embryonic stem cells (WiCell) and H7-derived downstream early mesoderm progenitors. The top 10% genes ranked by z-scores from the M3Drop package are retained.",
  BiocVersion = "3.7",
  Genome = "GRCh38",
  SourceType = "FASTQ",
  SourceUrl = "https://www.ncbi.nlm.nih.gov/sra?term=SRP073808",
  SourceVersion = NA,
  Species = "Homo sapiens",
  TaxonomyId = "9606",
  Coordinate_1_based = NA,
  DataProvider = "SRA (SRP073808), Koh et al. (2016)",
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "SingleCellExperiment",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/sce_filteredM3Drop10_KohTCC.rds",
  stringsAsFactors = FALSE
)

dfResFilteredM3Drop10KohTCC <- data.frame(
  Title = "clusteringSummaryFilteredM3Drop10KohTCC",
  Description = "Clustering results for sceFilteredM3Drop10KohTCC",
  BiocVersion = "3.7",
  Genome = "GRCh38",
  SourceType = NA,
  SourceUrl = NA,
  SourceVersion = NA,
  Species = "Homo sapiens",
  TaxonomyId = "9606",
  Coordinate_1_based = NA,
  DataProvider = NA,
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "data.frame",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/clustering_summary_filteredM3Drop10_KohTCC.rds",
  stringsAsFactors = FALSE
)

## ------------------------- filteredExpr10_Kumar --------------------------- ##
dfSCEFilteredExpr10Kumar <- data.frame(
  Title = "sceFilteredExpr10Kumar",
  Description = "Gene counts for scRNA-seq data set of mESCs with various genetic perturbations, cultured in different media. The top 10% genes with the highest average expression across all cells are retained.",
  BiocVersion = "3.7",
  Genome = "GRCm38",
  SourceType = "FASTQ",
  SourceUrl = "https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60749",
  SourceVersion = NA,
  Species = "Mus musculus",
  TaxonomyId = "10090",
  Coordinate_1_based = NA,
  DataProvider = "GEO (GSE60749), Kumar et al. (2014)",
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "SingleCellExperiment",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/sce_filteredExpr10_Kumar.rds",
  stringsAsFactors = FALSE
)

dfResFilteredExpr10Kumar <- data.frame(
  Title = "clusteringSummaryFilteredExpr10Kumar",
  Description = "Clustering results for sceFilteredExpr10Kumar",
  BiocVersion = "3.7",
  Genome = "GRCm38",
  SourceType = NA,
  SourceUrl = NA,
  SourceVersion = NA,
  Species = "Mus musculus",
  TaxonomyId = "10090",
  Coordinate_1_based = NA,
  DataProvider = NA,
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "data.frame",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/clustering_summary_filteredExpr10_Kumar.rds",
  stringsAsFactors = FALSE
)

## ------------------------- filteredHVG10_Kumar ---------------------------- ##
dfSCEFilteredHVG10Kumar <- data.frame(
  Title = "sceFilteredHVG10Kumar",
  Description = "Gene counts for scRNA-seq data set of mESCs with various genetic perturbations, cultured in different media. The top 10% most highly variable genes are retained.",
  BiocVersion = "3.7",
  Genome = "GRCm38",
  SourceType = "FASTQ",
  SourceUrl = "https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60749",
  SourceVersion = NA,
  Species = "Mus musculus",
  TaxonomyId = "10090",
  Coordinate_1_based = NA,
  DataProvider = "GEO (GSE60749), Kumar et al. (2014)",
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "SingleCellExperiment",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/sce_filteredHVG10_Kumar.rds",
  stringsAsFactors = FALSE
)

dfResFilteredHVG10Kumar <- data.frame(
  Title = "clusteringSummaryFilteredHVG10Kumar",
  Description = "Clustering results for sceFilteredHVG10Kumar",
  BiocVersion = "3.7",
  Genome = "GRCm38",
  SourceType = NA,
  SourceUrl = NA,
  SourceVersion = NA,
  Species = "Mus musculus",
  TaxonomyId = "10090",
  Coordinate_1_based = NA,
  DataProvider = NA,
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "data.frame",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/clustering_summary_filteredHVG10_Kumar.rds",
  stringsAsFactors = FALSE
)

## ------------------------ filteredM3Drop10_Kumar -------------------------- ##
dfSCEFilteredM3Drop10Kumar <- data.frame(
  Title = "sceFilteredM3Drop10Kumar",
  Description = "Gene counts for scRNA-seq data set of mESCs with various genetic perturbations, cultured in different media. The top 10% genes ranked by z-scores from the M3Drop package are retained.",
  BiocVersion = "3.7",
  Genome = "GRCm38",
  SourceType = "FASTQ",
  SourceUrl = "https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60749",
  SourceVersion = NA,
  Species = "Mus musculus",
  TaxonomyId = "10090",
  Coordinate_1_based = NA,
  DataProvider = "GEO (GSE60749), Kumar et al. (2014)",
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "SingleCellExperiment",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/sce_filteredM3Drop10_Kumar.rds",
  stringsAsFactors = FALSE
)

dfResFilteredM3Drop10Kumar <- data.frame(
  Title = "clusteringSummaryFilteredM3Drop10Kumar",
  Description = "Clustering results for sceFilteredM3Drop10Kumar",
  BiocVersion = "3.7",
  Genome = "GRCm38",
  SourceType = NA,
  SourceUrl = NA,
  SourceVersion = NA,
  Species = "Mus musculus",
  TaxonomyId = "10090",
  Coordinate_1_based = NA,
  DataProvider = NA,
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "data.frame",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/clustering_summary_filteredM3Drop10_Kumar.rds",
  stringsAsFactors = FALSE
)

## ----------------------- filteredExpr10_KumarTCC -------------------------- ##
dfSCEFilteredExpr10KumarTCC <- data.frame(
  Title = "sceFilteredExpr10KumarTCC",
  Description = "TCC counts for scRNA-seq data set of mESCs with various genetic perturbations, cultured in different media. The top 10% genes with the highest average expression across all cells are retained.",
  BiocVersion = "3.7",
  Genome = "GRCm38",
  SourceType = "FASTQ",
  SourceUrl = "https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60749",
  SourceVersion = NA,
  Species = "Mus musculus",
  TaxonomyId = "10090",
  Coordinate_1_based = NA,
  DataProvider = "GEO (GSE60749), Kumar et al. (2014)",
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "SingleCellExperiment",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/sce_filteredExpr10_KumarTCC.rds",
  stringsAsFactors = FALSE
)

dfResFilteredExpr10KumarTCC <- data.frame(
  Title = "clusteringSummaryFilteredExpr10KumarTCC",
  Description = "Clustering results for sceFilteredExpr10KumarTCC",
  BiocVersion = "3.7",
  Genome = "GRCm38",
  SourceType = NA,
  SourceUrl = NA,
  SourceVersion = NA,
  Species = "Mus musculus",
  TaxonomyId = "10090",
  Coordinate_1_based = NA,
  DataProvider = NA,
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "data.frame",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/clustering_summary_filteredExpr10_KumarTCC.rds",
  stringsAsFactors = FALSE
)

## ----------------------- filteredHVG10_KumarTCC --------------------------- ##
dfSCEFilteredHVG10KumarTCC <- data.frame(
  Title = "sceFilteredHVG10KumarTCC",
  Description = "TCC counts for scRNA-seq data set of mESCs with various genetic perturbations, cultured in different media. The top 10% most highly variable genes are retained.",
  BiocVersion = "3.7",
  Genome = "GRCm38",
  SourceType = "FASTQ",
  SourceUrl = "https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60749",
  SourceVersion = NA,
  Species = "Mus musculus",
  TaxonomyId = "10090",
  Coordinate_1_based = NA,
  DataProvider = "GEO (GSE60749), Kumar et al. (2014)",
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "SingleCellExperiment",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/sce_filteredHVG10_KumarTCC.rds",
  stringsAsFactors = FALSE
)

dfResFilteredHVG10KumarTCC <- data.frame(
  Title = "clusteringSummaryFilteredHVG10KumarTCC",
  Description = "Clustering results for sceFilteredHVG10KumarTCC",
  BiocVersion = "3.7",
  Genome = "GRCm38",
  SourceType = NA,
  SourceUrl = NA,
  SourceVersion = NA,
  Species = "Mus musculus",
  TaxonomyId = "10090",
  Coordinate_1_based = NA,
  DataProvider = NA,
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "data.frame",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/clustering_summary_filteredHVG10_KumarTCC.rds",
  stringsAsFactors = FALSE
)

## ---------------------- filteredM3Drop10_KumarTCC ------------------------- ##
dfSCEFilteredM3Drop10KumarTCC <- data.frame(
  Title = "sceFilteredM3Drop10KumarTCC",
  Description = "TCC counts for scRNA-seq data set of mESCs with various genetic perturbations, cultured in different media. The top 10% genes ranked by z-scores from the M3Drop package are retained.",
  BiocVersion = "3.7",
  Genome = "GRCm38",
  SourceType = "FASTQ",
  SourceUrl = "https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60749",
  SourceVersion = NA,
  Species = "Mus musculus",
  TaxonomyId = "10090",
  Coordinate_1_based = NA,
  DataProvider = "GEO (GSE60749), Kumar et al. (2014)",
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "SingleCellExperiment",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/sce_filteredM3Drop10_KumarTCC.rds",
  stringsAsFactors = FALSE
)

dfResFilteredM3Drop10KumarTCC <- data.frame(
  Title = "clusteringSummaryFilteredM3Drop10KumarTCC",
  Description = "Clustering results for sceFilteredM3Drop10KumarTCC",
  BiocVersion = "3.7",
  Genome = "GRCm38",
  SourceType = NA,
  SourceUrl = NA,
  SourceVersion = NA,
  Species = "Mus musculus",
  TaxonomyId = "10090",
  Coordinate_1_based = NA,
  DataProvider = NA,
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "data.frame",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/clustering_summary_filteredM3Drop10_KumarTCC.rds",
  stringsAsFactors = FALSE
)

## ----------------------- filteredExpr10_Trapnell -------------------------- ##
dfSCEFilteredExpr10Trapnell <- data.frame(
  Title = "sceFilteredExpr10Trapnell",
  Description = "Gene counts for scRNA-seq data set of primary myoblasts over a time course of serum-induced differentiation. The top 10% genes with the highest average expression across all cells are retained.",
  BiocVersion = "3.7",
  Genome = "GRCh38",
  SourceType = "FASTQ",
  SourceUrl = "https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE52529",
  SourceVersion = NA,
  Species = "Homo sapiens",
  TaxonomyId = "9606",
  Coordinate_1_based = NA,
  DataProvider = "GEO (GSE52529), Trapnell et al. (2014)",
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "SingleCellExperiment",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/sce_filteredExpr10_Trapnell.rds",
  stringsAsFactors = FALSE
)

dfResFilteredExpr10Trapnell <- data.frame(
  Title = "clusteringSummaryFilteredExpr10Trapnell",
  Description = "Clustering results for sceFilteredExpr10Trapnell",
  BiocVersion = "3.7",
  Genome = "GRCh38",
  SourceType = NA,
  SourceUrl = "",
  SourceVersion = NA,
  Species = "Homo sapiens",
  TaxonomyId = "9606",
  Coordinate_1_based = NA,
  DataProvider = NA,
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "data.frame",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/clustering_summary_filteredExpr10_Trapnell.rds",
  stringsAsFactors = FALSE
)

## ----------------------- filteredHVG10_Trapnell --------------------------- ##
dfSCEFilteredHVG10Trapnell <- data.frame(
  Title = "sceFilteredHVG10Trapnell",
  Description = "Gene counts for scRNA-seq data set of primary myoblasts over a time course of serum-induced differentiation. The top 10% most highly variable genes are retained.",
  BiocVersion = "3.7",
  Genome = "GRCh38",
  SourceType = "FASTQ",
  SourceUrl = "https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE52529",
  SourceVersion = NA,
  Species = "Homo sapiens",
  TaxonomyId = "9606",
  Coordinate_1_based = NA,
  DataProvider = "GEO (GSE52529), Trapnell et al. (2014)",
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "SingleCellExperiment",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/sce_filteredHVG10_Trapnell.rds",
  stringsAsFactors = FALSE
)

dfResFilteredHVG10Trapnell <- data.frame(
  Title = "clusteringSummaryFilteredHVG10Trapnell",
  Description = "Clustering results for sceFilteredHVG10Trapnell",
  BiocVersion = "3.7",
  Genome = "GRCh38",
  SourceType = NA,
  SourceUrl = "",
  SourceVersion = NA,
  Species = "Homo sapiens",
  TaxonomyId = "9606",
  Coordinate_1_based = NA,
  DataProvider = NA,
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "data.frame",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/clustering_summary_filteredHVG10_Trapnell.rds",
  stringsAsFactors = FALSE
)

## ---------------------- filteredM3Drop10_Trapnell ------------------------- ##
dfSCEFilteredM3Drop10Trapnell <- data.frame(
  Title = "sceFilteredM3Drop10Trapnell",
  Description = "Gene counts for scRNA-seq data set of primary myoblasts over a time course of serum-induced differentiation. The top 10% genes ranked by z-scores from the M3Drop package are retained.",
  BiocVersion = "3.7",
  Genome = "GRCh38",
  SourceType = "FASTQ",
  SourceUrl = "https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE52529",
  SourceVersion = NA,
  Species = "Homo sapiens",
  TaxonomyId = "9606",
  Coordinate_1_based = NA,
  DataProvider = "GEO (GSE52529), Trapnell et al. (2014)",
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "SingleCellExperiment",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/sce_filteredM3Drop10_Trapnell.rds",
  stringsAsFactors = FALSE
)

dfResFilteredM3Drop10Trapnell <- data.frame(
  Title = "clusteringSummaryFilteredM3Drop10Trapnell",
  Description = "Clustering results for sceFilteredM3Drop10Trapnell",
  BiocVersion = "3.7",
  Genome = "GRCh38",
  SourceType = NA,
  SourceUrl = "",
  SourceVersion = NA,
  Species = "Homo sapiens",
  TaxonomyId = "9606",
  Coordinate_1_based = NA,
  DataProvider = NA,
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "data.frame",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/clustering_summary_filteredM3Drop10_Trapnell.rds",
  stringsAsFactors = FALSE
)

## --------------------- filteredExpr10_TrapnellTCC ------------------------- ##
dfSCEFilteredExpr10TrapnellTCC <- data.frame(
  Title = "sceFilteredExpr10TrapnellTCC",
  Description = "TCC counts for scRNA-seq data set of primary myoblasts over a time course of serum-induced differentiation. The top 10% genes with the highest average expression across all cells are retained.",
  BiocVersion = "3.7",
  Genome = "GRCh38",
  SourceType = "FASTQ",
  SourceUrl = "https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE52529",
  SourceVersion = NA,
  Species = "Homo sapiens",
  TaxonomyId = "9606",
  Coordinate_1_based = NA,
  DataProvider = "GEO (GSE52529), Trapnell et al. (2014)",
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "SingleCellExperiment",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/sce_filteredExpr10_TrapnellTCC.rds",
  stringsAsFactors = FALSE
)

dfResFilteredExpr10TrapnellTCC <- data.frame(
  Title = "clusteringSummaryFilteredExpr10TrapnellTCC",
  Description = "Clustering results for sceFilteredExpr10TrapnellTCC",
  BiocVersion = "3.7",
  Genome = "GRCh38",
  SourceType = NA,
  SourceUrl = "",
  SourceVersion = NA,
  Species = "Homo sapiens",
  TaxonomyId = "9606",
  Coordinate_1_based = NA,
  DataProvider = NA,
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "data.frame",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/clustering_summary_filteredExpr10_TrapnellTCC.rds",
  stringsAsFactors = FALSE
)

## --------------------- filteredHVG10_TrapnellTCC -------------------------- ##
dfSCEFilteredHVG10TrapnellTCC <- data.frame(
  Title = "sceFilteredHVG10TrapnellTCC",
  Description = "TCC counts for scRNA-seq data set of primary myoblasts over a time course of serum-induced differentiation. The top 10% most highly variable genes are retained.",
  BiocVersion = "3.7",
  Genome = "GRCh38",
  SourceType = "FASTQ",
  SourceUrl = "https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE52529",
  SourceVersion = NA,
  Species = "Homo sapiens",
  TaxonomyId = "9606",
  Coordinate_1_based = NA,
  DataProvider = "GEO (GSE52529), Trapnell et al. (2014)",
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "SingleCellExperiment",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/sce_filteredHVG10_TrapnellTCC.rds",
  stringsAsFactors = FALSE
)

dfResFilteredHVG10TrapnellTCC <- data.frame(
  Title = "clusteringSummaryFilteredHVG10TrapnellTCC",
  Description = "Clustering results for sceFilteredHVG10TrapnellTCC",
  BiocVersion = "3.7",
  Genome = "GRCh38",
  SourceType = NA,
  SourceUrl = "",
  SourceVersion = NA,
  Species = "Homo sapiens",
  TaxonomyId = "9606",
  Coordinate_1_based = NA,
  DataProvider = NA,
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "data.frame",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/clustering_summary_filteredHVG10_TrapnellTCC.rds",
  stringsAsFactors = FALSE
)

## -------------------- filteredM3Drop10_TrapnellTCC ------------------------ ##
dfSCEFilteredM3Drop10TrapnellTCC <- data.frame(
  Title = "sceFilteredM3Drop10TrapnellTCC",
  Description = "TCC counts for scRNA-seq data set of primary myoblasts over a time course of serum-induced differentiation. The top 10% genes ranked by z-scores from the M3Drop package are retained.",
  BiocVersion = "3.7",
  Genome = "GRCh38",
  SourceType = "FASTQ",
  SourceUrl = "https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE52529",
  SourceVersion = NA,
  Species = "Homo sapiens",
  TaxonomyId = "9606",
  Coordinate_1_based = NA,
  DataProvider = "GEO (GSE52529), Trapnell et al. (2014)",
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "SingleCellExperiment",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/sce_filteredM3Drop10_TrapnellTCC.rds",
  stringsAsFactors = FALSE
)

dfResFilteredM3Drop10TrapnellTCC <- data.frame(
  Title = "clusteringSummaryFilteredM3Drop10TrapnellTCC",
  Description = "Clustering results for sceFilteredM3Drop10TrapnellTCC",
  BiocVersion = "3.7",
  Genome = "GRCh38",
  SourceType = NA,
  SourceUrl = NA,
  SourceVersion = NA,
  Species = "Homo sapiens",
  TaxonomyId = "9606",
  Coordinate_1_based = NA,
  DataProvider = NA,
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "data.frame",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/clustering_summary_filteredM3Drop10_TrapnellTCC.rds",
  stringsAsFactors = FALSE
)

## ---------------------- filteredExpr10_Zhengmix4eq ------------------------ ##
dfSCEFilteredExpr10Zhengmix4eq <- data.frame(
  Title = "sceFilteredExpr10Zhengmix4eq",
  Description = "UMI counts for scRNA-seq data set consisting of sorted B-cells, CD14 monocytes, naive cytotoxic and regulatory T-cells in equal proportions. The top 10% genes with the highest average expression across all cells are retained.",
  BiocVersion = "3.7",
  Genome = "hg19",
  SourceType = "TXT",
  SourceUrl = "https://support.10xgenomics.com/single-cell-gene-expression/datasets",
  SourceVersion = NA,
  Species = "Homo sapiens",
  TaxonomyId = "9606",
  Coordinate_1_based = NA,
  DataProvider = "10x Genomics",
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "SingleCellExperiment",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/sce_filteredExpr10_Zhengmix4eq.rds",
  stringsAsFactors = FALSE
)

dfResFilteredExpr10Zhengmix4eq <- data.frame(
  Title = "clusteringSummaryFilteredExpr10Zhengmix4eq",
  Description = "Clustering results for sceFilteredExpr10Zhengmix4eq",
  BiocVersion = "3.7",
  Genome = "hg19",
  SourceType = NA,
  SourceUrl = NA,
  SourceVersion = NA,
  Species = "Homo sapiens",
  TaxonomyId = "9606",
  Coordinate_1_based = NA,
  DataProvider = NA,
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "data.frame",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/clustering_summary_filteredExpr10_Zhengmix4eq.rds",
  stringsAsFactors = FALSE
)

## ---------------------- filteredHVG10_Zhengmix4eq ------------------------- ##
dfSCEFilteredHVG10Zhengmix4eq <- data.frame(
  Title = "sceFilteredHVG10Zhengmix4eq",
  Description = "UMI counts for scRNA-seq data set consisting of sorted B-cells, CD14 monocytes, naive cytotoxic and regulatory T-cells in equal proportions. The top 10% most highly variable genes are retained.",
  BiocVersion = "3.7",
  Genome = "hg19",
  SourceType = "TXT",
  SourceUrl = "https://support.10xgenomics.com/single-cell-gene-expression/datasets",
  SourceVersion = NA,
  Species = "Homo sapiens",
  TaxonomyId = "9606",
  Coordinate_1_based = NA,
  DataProvider = "10x Genomics",
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "SingleCellExperiment",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/sce_filteredHVG10_Zhengmix4eq.rds",
  stringsAsFactors = FALSE
)

dfResFilteredHVG10Zhengmix4eq <- data.frame(
  Title = "clusteringSummaryFilteredHVG10Zhengmix4eq",
  Description = "Clustering results for sceFilteredHVG10Zhengmix4eq",
  BiocVersion = "3.7",
  Genome = "hg19",
  SourceType = NA,
  SourceUrl = NA,
  SourceVersion = NA,
  Species = "Homo sapiens",
  TaxonomyId = "9606",
  Coordinate_1_based = NA,
  DataProvider = NA,
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "data.frame",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/clustering_summary_filteredHVG10_Zhengmix4eq.rds",
  stringsAsFactors = FALSE
)

## --------------------- filteredM3Drop10_Zhengmix4eq ----------------------- ##
dfSCEFilteredM3Drop10Zhengmix4eq <- data.frame(
  Title = "sceFilteredM3Drop10Zhengmix4eq",
  Description = "UMI counts for scRNA-seq data set consisting of sorted B-cells, CD14 monocytes, naive cytotoxic and regulatory T-cells in equal proportions. The top 10% genes ranked by z-scores from the M3Drop package are retained.",
  BiocVersion = "3.7",
  Genome = "hg19",
  SourceType = "TXT",
  SourceUrl = "https://support.10xgenomics.com/single-cell-gene-expression/datasets",
  SourceVersion = NA,
  Species = "Homo sapiens",
  TaxonomyId = "9606",
  Coordinate_1_based = NA,
  DataProvider = "10x Genomics",
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "SingleCellExperiment",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/sce_filteredM3Drop10_Zhengmix4eq.rds",
  stringsAsFactors = FALSE
)

dfResFilteredM3Drop10Zhengmix4eq <- data.frame(
  Title = "clusteringSummaryFilteredM3Drop10Zhengmix4eq",
  Description = "Clustering results for sceFilteredM3Drop10Zhengmix4eq",
  BiocVersion = "3.7",
  Genome = "hg19",
  SourceType = NA,
  SourceUrl = NA,
  SourceVersion = NA,
  Species = "Homo sapiens",
  TaxonomyId = "9606",
  Coordinate_1_based = NA,
  DataProvider = NA,
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "data.frame",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/clustering_summary_filteredM3Drop10_Zhengmix4eq.rds",
  stringsAsFactors = FALSE
)

## --------------------- filteredExpr10_Zhengmix4uneq ----------------------- ##
dfSCEFilteredExpr10Zhengmix4uneq <- data.frame(
  Title = "sceFilteredExpr10Zhengmix4uneq",
  Description = "UMI counts for scRNA-seq data set consisting of sorted B-cells, CD14 monocytes, naive cytotoxic and regulatory T-cells in unequal proportions. The top 10% genes with the highest average expression across all cells are retained.",
  BiocVersion = "3.7",
  Genome = "hg19",
  SourceType = "TXT",
  SourceUrl = "https://support.10xgenomics.com/single-cell-gene-expression/datasets",
  SourceVersion = NA,
  Species = "Homo sapiens",
  TaxonomyId = "9606",
  Coordinate_1_based = NA,
  DataProvider = "10x Genomics",
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "SingleCellExperiment",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/sce_filteredExpr10_Zhengmix4uneq.rds",
  stringsAsFactors = FALSE
)

dfResFilteredExpr10Zhengmix4uneq <- data.frame(
  Title = "clusteringSummaryFilteredExpr10Zhengmix4uneq",
  Description = "Clustering results for sceFilteredExpr10Zhengmix4uneq",
  BiocVersion = "3.7",
  Genome = "hg19",
  SourceType = NA,
  SourceUrl = NA,
  SourceVersion = NA,
  Species = "Homo sapiens",
  TaxonomyId = "9606",
  Coordinate_1_based = NA,
  DataProvider = NA,
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "data.frame",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/clustering_summary_filteredExpr10_Zhengmix4uneq.rds",
  stringsAsFactors = FALSE
)

## --------------------- filteredHVG10_Zhengmix4uneq ------------------------ ##
dfSCEFilteredHVG10Zhengmix4uneq <- data.frame(
  Title = "sceFilteredHVG10Zhengmix4uneq",
  Description = "UMI counts for scRNA-seq data set consisting of sorted B-cells, CD14 monocytes, naive cytotoxic and regulatory T-cells in unequal proportions. The top 10% most highly variable genes are retained.",
  BiocVersion = "3.7",
  Genome = "hg19",
  SourceType = "TXT",
  SourceUrl = "https://support.10xgenomics.com/single-cell-gene-expression/datasets",
  SourceVersion = NA,
  Species = "Homo sapiens",
  TaxonomyId = "9606",
  Coordinate_1_based = NA,
  DataProvider = "10x Genomics",
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "SingleCellExperiment",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/sce_filteredHVG10_Zhengmix4uneq.rds",
  stringsAsFactors = FALSE
)

dfResFilteredHVG10Zhengmix4uneq <- data.frame(
  Title = "clusteringSummaryFilteredHVG10Zhengmix4uneq",
  Description = "Clustering results for sceFilteredHVG10Zhengmix4uneq",
  BiocVersion = "3.7",
  Genome = "hg19",
  SourceType = NA,
  SourceUrl = NA,
  SourceVersion = NA,
  Species = "Homo sapiens",
  TaxonomyId = "9606",
  Coordinate_1_based = NA,
  DataProvider = NA,
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "data.frame",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/clustering_summary_filteredHVG10_Zhengmix4uneq.rds",
  stringsAsFactors = FALSE
)

## -------------------- filteredM3Drop10_Zhengmix4uneq ---------------------- ##
dfSCEFilteredM3Drop10Zhengmix4uneq <- data.frame(
  Title = "sceFilteredM3Drop10Zhengmix4uneq",
  Description = "UMI counts for scRNA-seq data set consisting of sorted B-cells, CD14 monocytes, naive cytotoxic and regulatory T-cells in unequal proportions. The top 10% genes ranked by z-scores from the M3Drop package are retained.",
  BiocVersion = "3.7",
  Genome = "hg19",
  SourceType = "TXT",
  SourceUrl = "https://support.10xgenomics.com/single-cell-gene-expression/datasets",
  SourceVersion = NA,
  Species = "Homo sapiens",
  TaxonomyId = "9606",
  Coordinate_1_based = NA,
  DataProvider = "10x Genomics",
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "SingleCellExperiment",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/sce_filteredM3Drop10_Zhengmix4uneq.rds",
  stringsAsFactors = FALSE
)

dfResFilteredM3Drop10Zhengmix4uneq <- data.frame(
  Title = "clusteringSummaryFilteredM3Drop10Zhengmix4uneq",
  Description = "Clustering results for sceFilteredM3Drop10Zhengmix4uneq",
  BiocVersion = "3.7",
  Genome = "hg19",
  SourceType = NA,
  SourceUrl = NA,
  SourceVersion = NA,
  Species = "Homo sapiens",
  TaxonomyId = "9606",
  Coordinate_1_based = NA,
  DataProvider = NA,
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "data.frame",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/clustering_summary_filteredM3Drop10_Zhengmix4uneq.rds",
  stringsAsFactors = FALSE
)

## ---------------------- filteredExpr10_Zhengmix8eq ------------------------ ##
dfSCEFilteredExpr10Zhengmix8eq <- data.frame(
  Title = "sceFilteredExpr10Zhengmix8eq",
  Description = "UMI counts for scRNA-seq data set consisting of sorted B-cells, naive cytotoxic, CD14 monocytes, regulatory T-cells, CD56 NK cells, memory T-cells, CD4 T-helper cells and naive T-cells in approximately equal proportions. The top 10% genes with the highest average expression across all cells are retained.",
  BiocVersion = "3.7",
  Genome = "hg19",
  SourceType = "TXT",
  SourceUrl = "https://support.10xgenomics.com/single-cell-gene-expression/datasets",
  SourceVersion = NA,
  Species = "Homo sapiens",
  TaxonomyId = "9606",
  Coordinate_1_based = NA,
  DataProvider = "10x Genomics",
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "SingleCellExperiment",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/sce_filteredExpr10_Zhengmix8eq.rds",
  stringsAsFactors = FALSE
)

dfResFilteredExpr10Zhengmix8eq <- data.frame(
  Title = "clusteringSummaryFilteredExpr10Zhengmix8eq",
  Description = "Clustering results for sceFilteredExpr10Zhengmix8eq",
  BiocVersion = "3.7",
  Genome = "hg19",
  SourceType = NA,
  SourceUrl = NA,
  SourceVersion = NA,
  Species = "Homo sapiens",
  TaxonomyId = "9606",
  Coordinate_1_based = NA,
  DataProvider = NA,
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "data.frame",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/clustering_summary_filteredExpr10_Zhengmix8eq.rds",
  stringsAsFactors = FALSE
)

## ---------------------- filteredHVG10_Zhengmix8eq ------------------------- ##
dfSCEFilteredHVG10Zhengmix8eq <- data.frame(
  Title = "sceFilteredHVG10Zhengmix8eq",
  Description = "UMI counts for scRNA-seq data set consisting of sorted B-cells, naive cytotoxic, CD14 monocytes, regulatory T-cells, CD56 NK cells, memory T-cells, CD4 T-helper cells and naive T-cells in approximately equal proportions. The top 10% most highly variable genes are retained.",
  BiocVersion = "3.7",
  Genome = "hg19",
  SourceType = "TXT",
  SourceUrl = "https://support.10xgenomics.com/single-cell-gene-expression/datasets",
  SourceVersion = NA,
  Species = "Homo sapiens",
  TaxonomyId = "9606",
  Coordinate_1_based = NA,
  DataProvider = "10x Genomics",
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "SingleCellExperiment",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/sce_filteredHVG10_Zhengmix8eq.rds",
  stringsAsFactors = FALSE
)

dfResFilteredHVG10Zhengmix8eq <- data.frame(
  Title = "clusteringSummaryFilteredHVG10Zhengmix8eq",
  Description = "Clustering results for sceFilteredHVG10Zhengmix8eq",
  BiocVersion = "3.7",
  Genome = "hg19",
  SourceType = NA,
  SourceUrl = NA,
  SourceVersion = NA,
  Species = "Homo sapiens",
  TaxonomyId = "9606",
  Coordinate_1_based = NA,
  DataProvider = NA,
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "data.frame",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/clustering_summary_filteredHVG10_Zhengmix8eq.rds",
  stringsAsFactors = FALSE
)

## --------------------- filteredM3Drop10_Zhengmix8eq ----------------------- ##
dfSCEFilteredM3Drop10Zhengmix8eq <- data.frame(
  Title = "sceFilteredM3Drop10Zhengmix8eq",
  Description = "UMI counts for scRNA-seq data set consisting of sorted B-cells, naive cytotoxic, CD14 monocytes, regulatory T-cells, CD56 NK cells, memory T-cells, CD4 T-helper cells and naive T-cells in approximately equal proportions. The top 10% genes ranked by z-scores from the M3Drop package are retained.",
  BiocVersion = "3.7",
  Genome = "hg19",
  SourceType = "TXT",
  SourceUrl = "https://support.10xgenomics.com/single-cell-gene-expression/datasets",
  SourceVersion = NA,
  Species = "Homo sapiens",
  TaxonomyId = "9606",
  Coordinate_1_based = NA,
  DataProvider = "10x Genomics",
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "SingleCellExperiment",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/sce_filteredM3Drop10_Zhengmix8eq.rds",
  stringsAsFactors = FALSE
)

dfResFilteredM3Drop10Zhengmix8eq <- data.frame(
  Title = "clusteringSummaryFilteredM3Drop10Zhengmix8eq",
  Description = "Clustering results for sceFilteredM3Drop10Zhengmix8eq",
  BiocVersion = "3.7",
  Genome = "hg19",
  SourceType = NA,
  SourceUrl = NA,
  SourceVersion = NA,
  Species = "Homo sapiens",
  TaxonomyId = "9606",
  Coordinate_1_based = NA,
  DataProvider = NA,
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "data.frame",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/clustering_summary_filteredM3Drop10_Zhengmix8eq.rds",
  stringsAsFactors = FALSE
)

## --------------------- filteredExpr10_SimKumar4easy ----------------------- ##
dfSCEFilteredExpr10SimKumar4easy <- data.frame(
  Title = "sceFilteredExpr10SimKumar4easy",
  Description = "Gene counts for simulated scRNA-seq data set with four subpopulations. The top 10% genes with the highest average expression across all cells are retained.",
  BiocVersion = "3.7",
  Genome = "GRCm38",
  SourceType = NA,
  SourceUrl = NA,
  SourceVersion = NA,
  Species = "Mus musculus",
  TaxonomyId = "10090",
  Coordinate_1_based = NA,
  DataProvider = NA,
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "SingleCellExperiment",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/sce_filteredExpr10_SimKumar4easy.rds",
  stringsAsFactors = FALSE
)

dfResFilteredExpr10SimKumar4easy <- data.frame(
  Title = "clusteringSummaryFilteredExpr10SimKumar4easy",
  Description = "Clustering results for sceFilteredExpr10SimKumar4easy",
  BiocVersion = "3.7",
  Genome = "GRCm38",
  SourceType = NA,
  SourceUrl = NA,
  SourceVersion = NA,
  Species = "Mus musculus",
  TaxonomyId = "10090",
  Coordinate_1_based = NA,
  DataProvider = NA,
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "data.frame",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/clustering_summary_filteredExpr10_SimKumar4easy.rds",
  stringsAsFactors = FALSE
)

## --------------------- filteredHVG10_SimKumar4easy ------------------------ ##
dfSCEFilteredHVG10SimKumar4easy <- data.frame(
  Title = "sceFilteredHVG10SimKumar4easy",
  Description = "Gene counts for simulated scRNA-seq data set with four subpopulations. The top 10% most highly variable genes are retained.",
  BiocVersion = "3.7",
  Genome = "GRCm38",
  SourceType = NA,
  SourceUrl = NA,
  SourceVersion = NA,
  Species = "Mus musculus",
  TaxonomyId = "10090",
  Coordinate_1_based = NA,
  DataProvider = NA,
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "SingleCellExperiment",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/sce_filteredHVG10_SimKumar4easy.rds",
  stringsAsFactors = FALSE
)

dfResFilteredHVG10SimKumar4easy <- data.frame(
  Title = "clusteringSummaryFilteredHVG10SimKumar4easy",
  Description = "Clustering results for sceFilteredHVG10SimKumar4easy",
  BiocVersion = "3.7",
  Genome = "GRCm38",
  SourceType = NA,
  SourceUrl = NA,
  SourceVersion = NA,
  Species = "Mus musculus",
  TaxonomyId = "10090",
  Coordinate_1_based = NA,
  DataProvider = NA,
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "data.frame",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/clustering_summary_filteredHVG10_SimKumar4easy.rds",
  stringsAsFactors = FALSE
)

## -------------------- filteredM3Drop10_SimKumar4easy ---------------------- ##
dfSCEFilteredM3Drop10SimKumar4easy <- data.frame(
  Title = "sceFilteredM3Drop10SimKumar4easy",
  Description = "Gene counts for simulated scRNA-seq data set with four subpopulations. The top 10% genes ranked by z-scores from the M3Drop package are retained.",
  BiocVersion = "3.7",
  Genome = "GRCm38",
  SourceType = NA,
  SourceUrl = NA,
  SourceVersion = NA,
  Species = "Mus musculus",
  TaxonomyId = "10090",
  Coordinate_1_based = NA,
  DataProvider = NA,
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "SingleCellExperiment",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/sce_filteredM3Drop10_SimKumar4easy.rds",
  stringsAsFactors = FALSE
)

dfResFilteredM3Drop10SimKumar4easy <- data.frame(
  Title = "clusteringSummaryFilteredM3Drop10SimKumar4easy",
  Description = "Clustering results for sceFilteredM3Drop10SimKumar4easy",
  BiocVersion = "3.7",
  Genome = "GRCm38",
  SourceType = NA,
  SourceUrl = NA,
  SourceVersion = NA,
  Species = "Mus musculus",
  TaxonomyId = "10090",
  Coordinate_1_based = NA,
  DataProvider = NA,
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "data.frame",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/clustering_summary_filteredM3Drop10_SimKumar4easy.rds",
  stringsAsFactors = FALSE
)

## --------------------- filteredExpr10_SimKumar4hard ----------------------- ##
dfSCEFilteredExpr10SimKumar4hard <- data.frame(
  Title = "sceFilteredExpr10SimKumar4hard",
  Description = "Gene counts for simulated scRNA-seq data set with four subpopulations. The top 10% genes with the highest average expression across all cells are retained.",
  BiocVersion = "3.7",
  Genome = "GRCm38",
  SourceType = NA,
  SourceUrl = NA,
  SourceVersion = NA,
  Species = "Mus musculus",
  TaxonomyId = "10090",
  Coordinate_1_based = NA,
  DataProvider = NA,
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "SingleCellExperiment",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/sce_filteredExpr10_SimKumar4hard.rds",
  stringsAsFactors = FALSE
)

dfResFilteredExpr10SimKumar4hard <- data.frame(
  Title = "clusteringSummaryFilteredExpr10SimKumar4hard",
  Description = "Clustering results for sceFilteredExpr10SimKumar4hard",
  BiocVersion = "3.7",
  Genome = "GRCm38",
  SourceType = NA,
  SourceUrl = NA,
  SourceVersion = NA,
  Species = "Mus musculus",
  TaxonomyId = "10090",
  Coordinate_1_based = NA,
  DataProvider = NA,
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "data.frame",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/clustering_summary_filteredExpr10_SimKumar4hard.rds",
  stringsAsFactors = FALSE
)

## --------------------- filteredHVG10_SimKumar4hard ------------------------ ##
dfSCEFilteredHVG10SimKumar4hard <- data.frame(
  Title = "sceFilteredHVG10SimKumar4hard",
  Description = "Gene counts for simulated scRNA-seq data set with four subpopulations. The top 10% most highly variable genes are retained.",
  BiocVersion = "3.7",
  Genome = "GRCm38",
  SourceType = NA,
  SourceUrl = NA,
  SourceVersion = NA,
  Species = "Mus musculus",
  TaxonomyId = "10090",
  Coordinate_1_based = NA,
  DataProvider = NA,
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "SingleCellExperiment",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/sce_filteredHVG10_SimKumar4hard.rds",
  stringsAsFactors = FALSE
)

dfResFilteredHVG10SimKumar4hard <- data.frame(
  Title = "clusteringSummaryFilteredHVG10SimKumar4hard",
  Description = "Clustering results for sceFilteredHVG10SimKumar4hard",
  BiocVersion = "3.7",
  Genome = "GRCm38",
  SourceType = NA,
  SourceUrl = NA,
  SourceVersion = NA,
  Species = "Mus musculus",
  TaxonomyId = "10090",
  Coordinate_1_based = NA,
  DataProvider = NA,
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "data.frame",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/clustering_summary_filteredHVG10_SimKumar4hard.rds",
  stringsAsFactors = FALSE
)

## -------------------- filteredM3Drop10_SimKumar4hard ---------------------- ##
dfSCEFilteredM3Drop10SimKumar4hard <- data.frame(
  Title = "sceFilteredM3Drop10SimKumar4hard",
  Description = "Gene counts for simulated scRNA-seq data set with four subpopulations. The top 10% genes ranked by z-scores from the M3Drop package are retained.",
  BiocVersion = "3.7",
  Genome = "GRCm38",
  SourceType = NA,
  SourceUrl = NA,
  SourceVersion = NA,
  Species = "Mus musculus",
  TaxonomyId = "10090",
  Coordinate_1_based = NA,
  DataProvider = NA,
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "SingleCellExperiment",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/sce_filteredM3Drop10_SimKumar4hard.rds",
  stringsAsFactors = FALSE
)

dfResFilteredM3Drop10SimKumar4hard <- data.frame(
  Title = "clusteringSummaryFilteredM3Drop10SimKumar4hard",
  Description = "Clustering results for sceFilteredM3Drop10SimKumar4hard",
  BiocVersion = "3.7",
  Genome = "GRCm38",
  SourceType = NA,
  SourceUrl = NA,
  SourceVersion = NA,
  Species = "Mus musculus",
  TaxonomyId = "10090",
  Coordinate_1_based = NA,
  DataProvider = NA,
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "data.frame",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/clustering_summary_filteredM3Drop10_SimKumar4hard.rds",
  stringsAsFactors = FALSE
)

## --------------------- filteredExpr10_SimKumar8hard ----------------------- ##
dfSCEFilteredExpr10SimKumar8hard <- data.frame(
  Title = "sceFilteredExpr10SimKumar8hard",
  Description = "Gene counts for simulated scRNA-seq data set with eight subpopulations. The top 10% genes with the highest average expression across all cells are retained.",
  BiocVersion = "3.7",
  Genome = "GRCm38",
  SourceType = NA,
  SourceUrl = NA,
  SourceVersion = NA,
  Species = "Mus musculus",
  TaxonomyId = "10090",
  Coordinate_1_based = NA,
  DataProvider = NA,
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "SingleCellExperiment",
  DispatchClass = "rds",
  RDataPath = "https://ndownloader.figshare.com/files/12384098",
  stringsAsFactors = FALSE
)

dfResFilteredExpr10SimKumar8hard <- data.frame(
  Title = "clusteringSummaryFilteredExpr10SimKumar8hard",
  Description = "Clustering results for sceFilteredExpr10SimKumar8hard",
  BiocVersion = "3.7",
  Genome = "GRCm38",
  SourceType = NA,
  SourceUrl = NA,
  SourceVersion = NA,
  Species = "Mus musculus",
  TaxonomyId = "10090",
  Coordinate_1_based = NA,
  DataProvider = NA,
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "data.frame",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/clustering_summary_filteredExpr10_SimKumar8hard.rds",
  stringsAsFactors = FALSE
)

## --------------------- filteredHVG10_SimKumar8hard ------------------------ ##
dfSCEFilteredHVG10SimKumar8hard <- data.frame(
  Title = "sceFilteredHVG10SimKumar8hard",
  Description = "Gene counts for simulated scRNA-seq data set with eight subpopulations. The top 10% most highly variable genes are retained.",
  BiocVersion = "3.7",
  Genome = "GRCm38",
  SourceType = NA,
  SourceUrl = NA,
  SourceVersion = NA,
  Species = "Mus musculus",
  TaxonomyId = "10090",
  Coordinate_1_based = NA,
  DataProvider = NA,
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "SingleCellExperiment",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/sce_filteredHVG10_SimKumar8hard.rds",
  stringsAsFactors = FALSE
)

dfResFilteredHVG10SimKumar8hard <- data.frame(
  Title = "clusteringSummaryFilteredHVG10SimKumar8hard",
  Description = "Clustering results for sceFilteredHVG10SimKumar8hard",
  BiocVersion = "3.7",
  Genome = "GRCm38",
  SourceType = NA,
  SourceUrl = NA,
  SourceVersion = NA,
  Species = "Mus musculus",
  TaxonomyId = "10090",
  Coordinate_1_based = NA,
  DataProvider = NA,
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "data.frame",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/clustering_summary_filteredHVG10_SimKumar8hard.rds",
  stringsAsFactors = FALSE
)

## -------------------- filteredM3Drop10_SimKumar8hard ---------------------- ##
dfSCEFilteredM3Drop10SimKumar8hard <- data.frame(
  Title = "sceFilteredM3Drop10SimKumar8hard",
  Description = "Gene counts for simulated scRNA-seq data set with eight subpopulations. The top 10% genes ranked by z-scores from the M3Drop package are retained.",
  BiocVersion = "3.7",
  Genome = "GRCm38",
  SourceType = NA,
  SourceUrl = NA,
  SourceVersion = NA,
  Species = "Mus musculus",
  TaxonomyId = "10090",
  Coordinate_1_based = NA,
  DataProvider = NA,
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "SingleCellExperiment",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/sce_filteredM3Drop10_SimKumar8hard.rds",
  stringsAsFactors = FALSE
)

dfResFilteredM3Drop10SimKumar8hard <- data.frame(
  Title = "clusteringSummaryFilteredM3Drop10SimKumar8hard",
  Description = "Clustering results for sceFilteredM3Drop10SimKumar8hard",
  BiocVersion = "3.7",
  Genome = "GRCm38",
  SourceType = NA,
  SourceUrl = NA,
  SourceVersion = NA,
  Species = "Mus musculus",
  TaxonomyId = "10090",
  Coordinate_1_based = NA,
  DataProvider = NA,
  Maintainer = "Charlotte Soneson <charlottesoneson@gmail.com>",
  RDataClass = "data.frame",
  DispatchClass = "rds",
  RDataPath = "DuoClustering2018/clustering_summary_filteredM3Drop10_SimKumar8hard.rds",
  stringsAsFactors = FALSE
)

## Combined meta-data
df_all <- rbind(
  dfSCEFilteredExpr10Koh,
  dfSCEFilteredHVG10Koh,
  dfSCEFilteredM3Drop10Koh,
  dfSCEFilteredExpr10KohTCC,
  dfSCEFilteredHVG10KohTCC,
  dfSCEFilteredM3Drop10KohTCC,
  dfSCEFilteredExpr10Kumar,
  dfSCEFilteredHVG10Kumar,
  dfSCEFilteredM3Drop10Kumar,
  dfSCEFilteredExpr10KumarTCC,
  dfSCEFilteredHVG10KumarTCC,
  dfSCEFilteredM3Drop10KumarTCC,
  dfSCEFilteredExpr10Trapnell,
  dfSCEFilteredHVG10Trapnell,
  dfSCEFilteredM3Drop10Trapnell,
  dfSCEFilteredExpr10TrapnellTCC,
  dfSCEFilteredHVG10TrapnellTCC,
  dfSCEFilteredM3Drop10TrapnellTCC,
  dfSCEFilteredExpr10Zhengmix4eq,
  dfSCEFilteredHVG10Zhengmix4eq,
  dfSCEFilteredM3Drop10Zhengmix4eq,
  dfSCEFilteredExpr10Zhengmix4uneq,
  dfSCEFilteredHVG10Zhengmix4uneq,
  dfSCEFilteredM3Drop10Zhengmix4uneq,
  dfSCEFilteredExpr10Zhengmix8eq,
  dfSCEFilteredHVG10Zhengmix8eq,
  dfSCEFilteredM3Drop10Zhengmix8eq,
  dfSCEFilteredExpr10SimKumar4easy,
  dfSCEFilteredHVG10SimKumar4easy,
  dfSCEFilteredM3Drop10SimKumar4easy,
  dfSCEFilteredExpr10SimKumar4hard,
  dfSCEFilteredHVG10SimKumar4hard,
  dfSCEFilteredM3Drop10SimKumar4hard,
  dfSCEFilteredExpr10SimKumar8hard,
  dfSCEFilteredHVG10SimKumar8hard,
  dfSCEFilteredM3Drop10SimKumar8hard,
  dfResFilteredExpr10Koh,
  dfResFilteredHVG10Koh,
  dfResFilteredM3Drop10Koh,
  dfResFilteredExpr10KohTCC,
  dfResFilteredHVG10KohTCC,
  dfResFilteredM3Drop10KohTCC,
  dfResFilteredExpr10Kumar,
  dfResFilteredHVG10Kumar,
  dfResFilteredM3Drop10Kumar,
  dfResFilteredExpr10KumarTCC,
  dfResFilteredHVG10KumarTCC,
  dfResFilteredM3Drop10KumarTCC,
  dfResFilteredExpr10Trapnell,
  dfResFilteredHVG10Trapnell,
  dfResFilteredM3Drop10Trapnell,
  dfResFilteredExpr10TrapnellTCC,
  dfResFilteredHVG10TrapnellTCC,
  dfResFilteredM3Drop10TrapnellTCC,
  dfResFilteredExpr10Zhengmix4eq,
  dfResFilteredHVG10Zhengmix4eq,
  dfResFilteredM3Drop10Zhengmix4eq,
  dfResFilteredExpr10Zhengmix4uneq,
  dfResFilteredHVG10Zhengmix4uneq,
  dfResFilteredM3Drop10Zhengmix4uneq,
  dfResFilteredExpr10Zhengmix8eq,
  dfResFilteredHVG10Zhengmix8eq,
  dfResFilteredM3Drop10Zhengmix8eq,
  dfResFilteredExpr10SimKumar4easy,
  dfResFilteredHVG10SimKumar4easy,
  dfResFilteredM3Drop10SimKumar4easy,
  dfResFilteredExpr10SimKumar4hard,
  dfResFilteredHVG10SimKumar4hard,
  dfResFilteredM3Drop10SimKumar4hard,
  dfResFilteredExpr10SimKumar8hard,
  dfResFilteredHVG10SimKumar8hard,
  dfResFilteredM3Drop10SimKumar8hard
)

## Save .csv file
write.csv(df_all, file = "../extdata/metadata.csv", row.names = FALSE)
